code/game/atoms.dm

```diff
@@ -1641,6 +1646,24 @@
 	if(ai_controller)
 		ai_controller = new ai_controller(src)
 
+///Setter for the "base_pixel_x" var to append behavior related to it's changing
+/atom/proc/set_base_pixel_x(var/new_value)
```

Comment:
ver-editing basepixels should also call this proc

Reply:
added to atoms.dm in 33979f0